vault-upgrade-hook/src/main/java/biz/netcentric/vlt/upgrade/util/PackageInstallLogger.java

```diff
@@ -1,90 +1,15 @@
-/*
- * (C) Copyright 2016 Netcentric AG.
- *
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- */
 package biz.netcentric.vlt.upgrade.util;
 
-import org.apache.jackrabbit.vault.fs.api.ProgressTrackerListener;
-import org.apache.jackrabbit.vault.fs.api.ProgressTrackerListener.Mode;
 import org.apache.jackrabbit.vault.packaging.InstallContext;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.slf4j.helpers.FormattingTuple;
-import org.slf4j.helpers.MessageFormatter;
 
-/**
- * Helper class to log during content package installation.
- */
-public class PackageInstallLogger {
-    private final Logger base;
+public interface PackageInstallLogger {
```

Comment:
What's the advantage of making this an Interface? In my opinion the Interface is just not needed overhead here. The "LoggerStub" class could even stay the same, also without interface.

Reply:
Without the interface, to replace logger implementation with a stub, they will need to share parent-child relationship what seems to be logically incorrect. 

With the interface, they do not share a direct relationship.

Reply:
Decision: remove interface.